### DISCUSSION TITLE
Game still detecting I am using multiple clients

### DISCUSSION THREAD

**Discussion by bsor-dev** [sandboxie-plus/Sandboxie#4213] [Q&A Feedback]
This program only allowed to use 2 client at a time. Thankfully, I managed to bypass it using sandboxie. However, inside the game the app still manage to detect that I am using multiple client. Is there any way or configuration I could set to prevent this? Thanks

**Comment by love-code-yeyixiao** [comment]
Maybe the game uses Denuvo, so it detects with your Steam Account and your System Unique Identity.












YeYixiao
***@***.***






---Original---
From: ***@***.***&gt;
Date: Sat, Sep 7, 2024 05:43 AM
To: ***@***.***&gt;;
Cc: ***@***.***&gt;;
Subject: [sandboxie-plus/Sandboxie] Game still detecting I am using multipleclients (Discussion #4213)




 
This program only allowed to use 2 client at a time. Thankfully, I managed to bypass it using sandboxie. However, inside the game the app still manage to detect that I am using multiple client. Is there any way or configuration I could set to prevent this? Thanks
